tests/protocol/test_datacontract.py

```diff
@@ -20,10 +20,10 @@
 from azurelinuxagent.common.datacontract import get_properties, set_properties, DataContract, DataContractList
 
 
-class SampleDataContract(DataContract):
+class SampleDataContract(DataContract): # pylint: disable=too-few-public-methods
     def __init__(self):
-        self.foo = None
-        self.bar = DataContractList(int)
+        self.foo = None # pylint: disable=blacklisted-name
```

Comment:
I couldn't find a definition for this, could you add it please! Thanks!

Reply:
This error is only reported in python{2.7,3.4}, so I only put the definition in `2.6.pylintrc`.

I'm assuming you looked in `3.6.pylintrc` and couldn't find it?

Reply:
You mean the `2.7.pylintrc`? I looked at both (for this branch) and couldn't find it. Is it part of some other branch?

Reply:
Sorry, yes, typo.

I'm seeing it on line 13 in that file, both on my local branch, and in this PR. Maybe you're not looking at all commits?

Reply:
Ahh got it, I was in the wrong branch. Thanks for confirming :)